documentation/Using-OVS-DPDK-and-COR.md

```diff
@@ -0,0 +1,82 @@
+# Setup for Running OVS-DPDK
+
+## Installation of DPDK and OVS on host system
+
+There is tight coupling between OVS and DPDK versioning, and what is available via distributions is often very out of date.  The directions below will instruct you how to build and configure from source.  As a prequisite to building, you'll need a few packages on your host.  As an example, on an Ubuntu host system you'd need to install as follows: 
```

Comment:
Well, in this case, I'm noting the tight coupling between the two projects and asking them to install a particular version of each which are known working.  In essence, I'm saying "please do not go ahead and check for latest and upgrade"